### ISSUE TITLE
Do not fetch replies to comments w/o any replies

### ISSUE THREAD

**Issue by Ascor8522** [LLK/scratch-www#5134]
### Steps to Reproduce

1. Go on the [scratch website](https://scratch.mit.edu).
2. Go on a project with many comments. Any project featured on the home page will do.
3. Notice how the (first 20) comments are loaded. (from the API, at `https://api.scratch.mit.edu/users/$USER/projects/$PROJECT_ID/comments?offset=0&limit=20`)
Some comments might no have any replies to them. (`reply_count` proprety set to `0`)
4. However, the replies for those comments are still fetched from the API.
These requests are unnecessary and can slow down the browser, making it wait for dozen of requests to complete when not needed.

Only the replies of the comments with 1 or more replies should be fetched.

I believe this line is responsible for that.
https://github.com/LLK/scratch-www/blob/ecd4fa9438fd9ba0d2e1a5741964cbec17d63791/src/redux/project-comment-actions.js#L72

An easy fix would be to filter the array of comments just recieved, based on how many replies they have.
```js
dispatch(getReplies(id, body.filter(comment => comment.reply_count > 0).map(comment => comment.id), 0, ownerUsername, isAdmin, token)); 
```

### Operating System and Browser

Any

**Comment by Naleksuh** [comment]
@BryceLTaylor , I see this is still an issue after 2-3 months. Would you mind if I fixed this?

**Comment by AmazingMech2418** [comment]
It might be even more efficient to send the first 20 replies per thread with the original request to prevent excessive requests, and then send the rest through further requests when the user clicks to load more replies. It also would be possible to load all replies along with the original comments, but that might lead to excessive data per request as was mentioned in a comment in #5537 .

**Comment by jeffalo** [comment]
Looks like this is being resolved by #5630

**Comment by paulkaplan** [comment]
FWIW the reason for the individual requests is that they are more cacheable at the CDN layer and so create less of a backend server burden (which is a bit counterintuitive, since there are more requests).
